releasenotes/notes/aqgd-epochs-8e6ad87971e8e2db.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - Extension to the previous aqgd classical optimizer with the 
```

Comment:
Could we be more specific about that the actual change(s) to AQGD as it existed rather than text that describes AQGD more generally that can equally well apply to the current version.

Reply:
@pbark Please take a look.

Reply:
updated the release not to specify what is new from the description